<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HP" name="Setup_23: " tests="1" failures="1" timestamp="24 Nov 2015 09:16:53 GMT" time="9.629" errors="0">
  <testcase name="firstBlockOFQuestionSecFunctionality" time="0.005" classname="com.zinghr.tna.index.SettingsTimeAttendanceSetupIndex">
    <failure type="java.lang.Error" message="Unresolved compilation problems: 
	cmp_code cannot be resolved to a variable
	emp_code cannot be resolved to a variable
	pswd cannot be resolved to a variable
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
	cmp_code cannot be resolved to a variable
	emp_code cannot be resolved to a variable
	pswd cannot be resolved to a variable

	at com.zinghr.login.indexPage.LoginIndexPage.loginSuccessfully(LoginIndexPage.java:140)
	at com.zinghr.tna.index.SettingsTimeAttendanceSetupIndex.firstBlockOFQuestionSecFunctionality(SettingsTimeAttendanceSetupIndex.java:384)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- firstBlockOFQuestionSecFunctionality -->
</testsuite> <!-- Setup_23:  -->
